lib/manageiq/api/common/graphql/generator.rb

```diff
@@ -60,7 +60,7 @@ def self.collection_field_resolvers(schema_overlay, collection)
           end
 
           def self.init_schema(request, schema_overlay = {})
-            api_version       = ::ManageIQ::API::Common::GraphQL.version(request)
+            api_version       = GraphQL.version(request)
```

Comment:
Is this now referencing `::GraphQL` in the `graphql` gem?  If so, can https://github.com/ManageIQ/manageiq-api-common/pull/78/files#diff-bb4277a3c7abc4a5d84ffa1c301b5048R16 be removed?  If not, I think we should be explicit to avoid constant problems.

Reply:
was our own version, had updated for readability, but no problem, going back to fully qualified. 👍